docs/documentation/pages/DECKHOUSE-FAQ.md

```diff
@@ -463,6 +463,12 @@ This feature is available in Enterprise Edition only.
    `dhctl mirror` supports digesting of the final set of Deckhouse images with the GOST R 34.11-2012 (Stribog) hash function (the `--gost-digest` parameter).
    The checksum will be logged and written to a file with the `.tar.gostsum` extension next to the tar-archive containing the Deckhouse images.
 
+   Starting from version 1.59.1, `dhctl mirror` supports splitting the images bundle into chunks of arbitrary size instead of dumping all images into a single tar file.
```

Comment:
```suggestion
   Starting from version 1.59.0, `dhctl mirror` supports splitting the images bundle into chunks of arbitrary size instead of dumping all images into a single tar file.
```